Source/JavaScriptCore/runtime/GlobalObjectMethodTable.h

```diff
@@ -45,6 +45,8 @@ enum class JSPromiseRejectionOperation : unsigned {
     Handle, // When a handler is added to a rejected promise for the first time.
 };
 
+enum class CompilationType;
```

Comment:
This forward declaration should be sharing a paragraph with the other forward declaration, of `ScriptExecutionStatus`, not after the declaration of `JSPromiseRejectionOperation`.